libs/documentation/src/lib/components/formly-checkbox/demos/checkbox-radio/checkbox-radio.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+
+@Component({
+  selector: 'gsa-sam-checkbox-radio',
+  templateUrl: './checkbox-radio.component.html',
+  styleUrls: ['./checkbox-radio.component.scss'],
+})
+export class CheckboxRadio {
+  form = new FormGroup({});
+  model: any = {};
+  options: FormlyFormOptions = {};
+  fields: FormlyFieldConfig[] = [
+    {
+      wrappers: ['panel'],
```

Comment:
There may be some confusion, the panel wrapper example was to show how fieldgroup can be used to add a layer to apply CSS. We should try to make this work without the complication of nested forms if possible, something with fieldgroup and css would be ideal.